src/Http/Http.Abstractions/src/StatusCodes.cs

```diff
@@ -283,6 +283,11 @@ public static class StatusCodes
     /// </summary>
     public const int Status451UnavailableForLegalReasons = 451;
 
+    /// <summary>
+    /// HTTP status code 499.
+    /// </summary>
+    public const int Status499ClientConnectionClosed = 499;
```

Comment:
nginx named it `ClientClosedRequest`:
https://github.com/nginx/nginx/blob/bfc5b35827903a3c543b58e4562db8b62021c164/src/http/ngx_http_request.h#L128-L134